ruby/vampires.rb

```diff
@@ -0,0 +1,77 @@
+attempts = 0
+
+until attempts == 4
+	attempts += 1
+
+puts "What is your name?"
+name = gets.chomp
+	
+
+def age_finder(x, y)
+		 (2016 - y.to_i) == x.to_i
+end
+
+puts "How old are you? What year were you born?"
+age = gets.chomp
+year = gets.chomp
+
+	
+puts "The cafeteria serves garlic bread. Should we order some for you?"
+garlic_bread = gets.chomp
+
+	if garlic_bread == "yes"
+	  garlic_bread = true
+	else
+		garlic_bread == "no"
+	  garlic_bread = false
+	end
+
+puts "Would you like to enroll in the company's health insurance?"
+insurance = gets.chomp
+
+	if insurance == "yes"
+	  insurance = true
+	elsif insurance == "no"
+	  insurance = false
+	else insurance == nil
+		insurance = false
+	end
+
+
+if age_finder(age.to_i, year.to_i) && (garlic_bread || insurance)
```

Comment:
Surely we can't just puts out the result otherwise our screen would be full of each vampire check that passed. What we could do is keep setting a variable result within each if block and then at the end of all the blocks display the result to look something like this
```
if condition_1
	result = 'low'
end

if condition_2
	result = 'medium'
end

if condition_3
	result = 'high'
end

puts result
```